docs/source/CartesianConfig.rst

```diff
@@ -193,14 +196,75 @@ For example:
 
     only default
 
-Results in the following outcome:
+Results in the following:
 
 ::
 
-    name = default.three.one
-    key1 = Hello
-    key2 =
-    key3 = World
+    dict 1: default.three.one
+      key1 = Hello
+      key2 =
+      key3 = World
+
+
+2. exclude all combinations names not matching a pattern. Requires keyword 'no'
+Useful to remove particular conflicting key/value pairs from some combinations.
+
+Example::
+
+    key1 = value1
+    key2 = value2
+
+    variants:
+        - one:
+            key1 = Hello World
+            key2 = foo1
+        - two:
+            key2 = foo2
+        - three:
+
+    variants:
+        - A:
+            no one
```

Comment:
The `variant A` is not really necessary and only makes it harder to follow. How about:

```
...
        - three:

no one
```

which creates 2 variants `two` and `three`?